pymbolic/mapper/equalizer.py

```diff
@@ -0,0 +1,169 @@
+import pymbolic
+
+
+class EqualizerMapper(pymbolic.mapper.Mapper):
```

Comment:
I wonder about the name. `Equalizer` (as in audio use) is something that *makes* things the same. `EqualityComparer` or `EqualityComparisonMapper` are clunky but IMO better.

(Also affects the file name.)